extensions/istio/generator-v1alpha3/go.mod

```diff
@@ -0,0 +1,17 @@
+module github.com/fabric8io/kubernetes-client/extensions/istio/generator
+
+go 1.13
+
+require (
+	github.com/fabric8io/kubernetes-client/generator v0.0.0
+	github.com/gogo/protobuf v1.3.2
+	istio.io/api v0.0.0-20211111170902-f2972da3100e
+	istio.io/client-go v1.12.0-alpha.5.0.20211111180919-207fea46070b
```

Comment:
Can we use the released version, instead, please?

Reply:
Updated